internal/bundle/python_tasks_test.go

```diff
@@ -0,0 +1,255 @@
+package bundle
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"slices"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/databricks/cli/bundle/run/output"
+	"github.com/databricks/cli/internal"
+	"github.com/databricks/cli/libs/filer"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+	"github.com/stretchr/testify/require"
+)
+
+const PY_CONTENT = `# Databricks notebook source
+import os
+import sys
+import json
+
+out = {"PYTHONPATH": sys.path, "CWD": os.getcwd()}
+json_object = json.dumps(out, indent = 4)
+dbutils.notebook.exit(json_object)
+`
+
+const SPARK_PY_CONTENT = `
+import os
+import sys
+import json
+
+out = {"PYTHONPATH": sys.path, "CWD": os.getcwd()}
+json_object = json.dumps(out, indent = 4)
+print(json_object)
+`
+
+type testOutput struct {
+	PythonPath []string `json:"PYTHONPATH"`
+	Cwd        string   `json:"CWD"`
+}
+
+type testFiles struct {
+	w               *databricks.WorkspaceClient
+	pyNotebookPath  string
+	sparkPythonPath string
+	wheelPath       string
+}
+
+type testOpts struct {
+	name                    string
+	includeNotebookTasks    bool
+	includeSparkPythonTasks bool
+	includeWheelTasks       bool
+	wheelSparkVersions      []string
+}
+
+func TestAccRunPythonTaskWorkspace(t *testing.T) {
+	// TODO: remove RUN_PYTHON_TASKS_TEST when ready to be executed as part of nightly
+	internal.GetEnvOrSkipTest(t, "RUN_PYTHON_TASKS_TEST")
```

Comment:
there's `ucws` testing environment, which tests against UC-enabled workspace, use those env vars

Reply:
atm we don't plan to execute it on continuously  basis yet, just as a script to be executed manually to generate the data

Reply:
yep, but you can still execute it with `deco env shell` and pick a `*-ucws` environment